doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -449,12 +449,18 @@ Fixing a commit message:
 
     git commit --amend
 
-Un-do the last commit (leaving changed files in your working directory)
+Un-do the last commit (leaving changed files in your working directory):
 
 .. code-block:: bash
 
     git reset --soft HEAD~1
 
+Reapplying changes from the current branch onto an updated version of master:
+
+.. code-block:: bash
+
+    git rebase master
```

Comment:
Let's make it be
```
git fetch upstream
git rebase upstream/master
```

Reply:
(we continued conversation after the initial "looks good" and decided this would be useful too)